theme/templates/base.html

```diff
@@ -78,31 +78,71 @@ <h2 class="text-white">sciwork</h2>
   {% block content %}{% endblock %}
 </main>
 
-<footer class="items-center bg-gray-800 text-white grow">
-  <div class="container mx-auto max-w-6xl px-5 py-2">
-    <h3>contact us</h3>
-    <div class="py-4 text-xl">
-    <a class="fa-stack" target="_blank" href="https://twitter.com/intent/tweet?screen_name=sciwork&ref_src=twsrc%5Etfw" data-show-count="false">
-      <i class="fas fa-circle fa-stack-2x text-white"></i>
-      <i class="fab fa-twitter fa-stack-1x text-gray-800" style="padding: 0.1rem 0 0 0.1rem"></i>
-    </a>
-    <a class="fa-stack" target="_blank" href="mailto: contact@sciwork.dev">
-      <i class="fas fa-circle fa-stack-2x text-white"></i>
-      <i class="fas fa-envelope fa-stack-1x grey-text text-gray-800"></i>
-    </a>
-    <a class="fa-stack" target="_blank" href="https://github.com/sciwork/swportal">
-      <i class="fas fa-circle fa-stack-2x text-white"></i>
-      <i class="fab fa-github fa-stack-1x grey-text text-gray-800"></i>
-    </a>
+
+<footer class="bg-blue-900 text-white">
```

Comment:
I think we can extend the width of the footer, it looks a little cramped on my screen

<img width="1211" alt="image" src="https://github.com/user-attachments/assets/99e1b572-7ec2-4400-8a52-c62248284871">

Reply:
Do you mean making every section more horizontally distributed??
![footer](https://github.com/user-attachments/assets/666003f4-0ae1-4bd8-8d93-b5ac5afbe2c2)

Reply:
I think it looks better now!